astropy/time/tests/test_basic.py

```diff
@@ -2102,3 +2102,10 @@ def test_ymdhms_output():
 def test_broadcasting_writeable():
     t = Time('J2015') + np.linspace(-1, 1, 10)*u.day
     t[2] = Time(58000, format="mjd")
+
+
+def test_repr_large_array():
+    t = Time(np.linspace(2458780, 2458790, 100), format='jd')
```

Comment:
This test can be made a bit cleaner by using `t = Time(np.arange(100), format='mjd')`, which gives: `<Time object: scale='utc' format='mjd' value=[ 0.  1.  2. ... 97. 98. 99.]>`